lib/node_modules/@stdlib/plot/table/unicode/lib/props/row-separator/set.js

```diff
@@ -0,0 +1,59 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var isString = require( '@stdlib/assert/is-string' );
+var format = require( '@stdlib/string/format' );
+
+
+// VARIABLES //
+
+var CHARACTER_LENGTH = 1;
+
+
+// MAIN //
+
+/**
+* Sets the row separator character.
+*
+* @private
+* @param {string} separator - row separator
+* @throws {TypeError} must be a character string or 'None'
+*/
+function set( separator ) {
+	/* eslint-disable no-invalid-this */
+	if ( !isString( separator ) ) {
+		throw new TypeError( format( 'invalid assignment. `%s` must be a string. Value: `%s`.', 'rowSeparator', separator ) );
+	}
+	if ( separator === 'None' ) {
```

Comment:
I wouldn't make `None` the sentinel.

Reply:
Do you mean we should another value to denote `None` (like `null` or `undefined`)?

Reply:
Yes, or the empty string.

Reply:
You should not use `undefined` as the sentinel.